deltachat-ios/DC/DcAccount.swift

```diff
@@ -12,6 +12,7 @@ public class DcAccounts {
     public var fetchSemaphore: DispatchSemaphore?
 
     private var encryptedDatabases: [Int: Bool] = [:]
+    private var justAdded: [Int: Bool] = [:]
```

Comment:
Instead of a dictionary, what do you think about using a simple list `freshlyAddedAccountIds: [Int] = []` as the `Bool` will never be `false`?

Reply:
totally makes sense and makes the gist clearer (i was just copying the pattern from `encryptedDatabases` databases without thinking much about it :)

i added a pr that picks up the idea